lib/srv/termmanager.go

```diff
@@ -221,6 +220,8 @@ func (g *TermManager) AddReader(name string, r io.Reader) {
 			if err != nil {
 				log.Warnf("Failed to read from remote terminal: %v", err)
 				g.DeleteReader(name)
+				// close session if any peer disconnects
+				g.Close()
```

Comment:
@zmb3 @xacrimon 

I decided to close the session if any peer disconnects, however in theory we can continue the session if there are multiple peers.

Reply:
@tigrato I don't think we can do this. Moderated sessions relies on the fact that people can come and go, the session should only close if it cannot continue per a require policy. This would be a terribly breaking change that would disrupt current customer workflows. Can we work around this?